mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -313,6 +293,29 @@ void CLeaderboardsTimes::FillLeaderboards(bool bFullUpdate)
     }
 }
 
+void CLeaderboardsTimes::LocalTimesVectorToLeaderboards()
+{
+    SetVisible(false);
+    KeyValuesAD kvLocalTimes("local");
+    ConvertLocalTimes(kvLocalTimes);
+    if (kvLocalTimes && !kvLocalTimes->IsEmpty())
```

Comment:
Since you now create the local times KV you can get rid of this null check and early return:
```suggestion
    if (kvLocalTimes->IsEmpty())
        return;
```